=== modified file 'web/static/src/js/views.js'
--- web/static/src/js/views.js	2014-02-17 10:49:20 +0000
+++ web/static/src/js/views.js	2014-03-11 11:43:46 +0000
@@ -1217,7 +1217,7 @@
         this.dataset = dataset;
         this.model_id = model_id;
         if (args && args[0].error) {
-            this.do_warn(_t('Uploading Error'), args[0].error);
+            alert(_t('Uploading Error')+ args[0].error);
         }
         if (!model_id) {
             this.on_attachments_loaded([]);

